metacat-common-server/src/main/java/com/netflix/metacat/common/server/connectors/ConnectorContext.java

```diff
@@ -26,7 +28,9 @@
  * @author tgianos
  */
 @Data
+@NoArgsConstructor
+@AllArgsConstructor
 public class ConnectorContext {
-    private final long timestamp;
-    private final String userName;
+    private long timestamp;
```

Comment:
Dozer was not able to map MetacatRequestContext to ConnectorContext if it does not have an default empty constructor. We can define factories within dozer to overcome the problem.